doc/releases/changelog-dev.md

```diff
@@ -148,9 +148,12 @@
 * Implemented kwargs (`check_interface`, `check_trainability`, `rtol` and `atol`) support in `qml.equal` for the operators `Pow`, `Adjoint`, `Exp`, and `SProd`.
   [(#5668)](https://github.com/PennyLaneAI/pennylane/issues/5668)
   
-* ``qml.QutritDepolarizingChannel`` has been added, allowing for depolarizing noise to be simulated on the `default.qutrit.mixed` device.
+* `qml.QutritDepolarizingChannel` has been added, allowing for depolarizing noise to be simulated on the `default.qutrit.mixed` device.
   [(#5502)](https://github.com/PennyLaneAI/pennylane/pull/5502)
 
+* `qml.TritFlip` has been added, allowing for parameter flipping errors, such as misclassification, to be simulated on the `default.qutrit.mixed` device.
```

Comment:
The language "parameter flipping" is used in a couple instances in this PR, but "parameter" in the context of PennyLane generally refers to gate parameters (i.e., rotation angles). Could we change the language to clarify what this means?

Reply:
Definitely, I was trying to come up with a good way to explain what I was thinking, but I fumbled.